Created glossary.md and defined project brand related terms (#326)
notaryproject/notaryproject.dev: content/en/docs/glossary.md

---
title: Glossary
description: A list of Notary Project's terminologies 
weight: 11
---
### Notary 

> Notary is a TUF [(The Update Framework)](https://github.com/theupdateframework/specification) based artifact signing solution. It consists of both [server](https://github.com/notaryproject/notary/blob/master/cmd/notary-server) and [client](https://github.com/notaryproject/notary/blob/master/cmd/notary) components for running and interacting with trusted collections. The client component is referred to as the Notary command-line interface (CLI), which handles tasks such as signing and pushing container images to a registry and updating metadata. The CLI communicates with the registry and the key and metadata management server component.

Reviewer: ```suggestion
Notary is a TUF [(The Update Framework)](https://theupdateframework.com/) based artifact signing solution. It consists of both server and client components. The server infrastructure tightly integrates with the container registry and keeps track of the images pushed to the registry. The client component is referred to as the notary command-line interface (CLI), which handles tasks such as signing and pushing container images to a registry and updating metadata. The CLI communicates with the registry and the key and metadata management server component. The most prominent use of this implementation is in Docker Content Trust (DCT). The server and the client implementation can be found in the [notary](https://github.com/notaryproject/notary) repository under the Notary Project organization.
```

### Notation

> Notation is a non-TUF-based artifact signing solution. It refers to the implementation of the Notation CLI and the Notation Go libraries. Notation offers signing and verification workflows for OCI artifacts, enables signature portability across OCI-compliant registries, and facilitates integration with 3rd party key management solutions through a plugin model. The CLI itself is referred to as the "Notation CLI," while the libraries are referrred to as the [Notation-Go](https://github.com/notaryproject/notation-go) library and the [Notation-Core-Go](https://github.com/notaryproject/notation-core-go) library. These libraries can be used independently of the Notation CLI.

Reviewer: ```suggestion
Notation is a non-TUF-based artifact signing solution. It refers to the implementation of the Notation CLI and the Notation libraries. Notation offers signing and verification workflows for OCI artifacts, enables signature portability across OCI-compliant registries, and facilitates integration with third party key management solutions through a plugin model. The CLI itself is referred to as the [Notation CLI](https://github.com/notaryproject/notation), while the libraries are referred to as the [Notation-Go](https://github.com/notaryproject/notation-go) library and the [Notation-Core-Go](https://github.com/notaryproject/notation-core-go) library. These libraries can be used independently of the Notation CLI.
```

> ### Notary Project Specifications

Reviewer 2: I think we need to explain a dedicated term `Notary Project` in the page.

Author: I have included the Notary Project term in the glossary and provided an explanation for it.

> Notary Project Specifications are a collection of specifications distributed across "subprojects" within the Notary Project. These specifications are also utilized by other OSS projects and/or vendor tools that seek to interoperate with the Notary project tooling.

Reviewer: ```suggestion
Notary Project Specifications are a collection of specifications distributed across "subprojects" within the Notary Project. These specifications are also utilized by other OSS projects and/or vendor tools that seek to interoperate with the Notary project tooling. These specifications are stored in the [notaryproject](https://github.com/notaryproject/notaryproject) repository under the Notary Project.
```

Notary Project specifications are categorized into:

**OCI Signature Specifications:**

> These specifications are specific to the Open Container Initiative (OCI). It describes how the Notary Project's Signing Scheme is applied to signatures stored in OCI registries. This category includes the following specifications:

Reviewer: ```suggestion
These specifications are specific to the [Open Container Initiative (OCI)](https://github.com/opencontainers). It describes how the Notary Project's Signing Scheme is applied to signatures stored in OCI registries. This category includes the following specifications:
```

> - [Notary Project OCI Signature Specification](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-specification.md)

Reviewer 2: The `main` branch is a development branch so the content may be changed anytime. These URLs should be pointed to a specific tag (i.e. v1.0.0) after the `notaryproject` released v1.0.0 as described in https://github.com/notaryproject/notaryproject/issues/261.

Author: Acknowledged, I'll ensure to point the URLs to the specific tag (v1.0.0) after the `notaryproject` releases  v1.0.0.

- [Notary Project OCI Signing and Verification Workflow](https://github.com/notaryproject/notaryproject/blob/main/specs/signing-and-verification-workflow.md)
- [Notary Project OCI COSE Envelope](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-envelope-cose.md)
- [Notary Project OCI JWS Envelope](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-envelope-jws.md)

**Signing Scheme Specification:** The Signing Scheme Specification is not specific to any particular storage method. It describes the supported signing schemes. Any tool that supports those schemes might be able to produce signatures that are compatible with Notary Project tools. The specification under this category includes the [Notary Project Signing Scheme.](https://github.com/notaryproject/notaryproject/blob/main/specs/signing-scheme.md)
### TUF

> The Update Framework(TUF) is a project under the Notary Project that aims to implement the full TUF specification in a registry-native way. The initial version of Notary did not support transferring signatures between different registries and did not gain widespread adoption. The TUF project aims to develop an enhanced version of Notary, addressing these limitations and encouraging widespread adoption and usage.

Reviewer: ```suggestion
TUF refers to [The Update Framework[(https://theupdateframework.com/). A [tuf](https://github.com/notaryproject/tuf) repository under the Notary Project organization is intended for prototyping the storage of TUF metadata in OCI-compliant registries. It is not under active development at the moment.
```

---
title: Glossary
description: A list of Notary Project's terminologies 
weight: 11
---

> ### Notary Project 

Reviewer 3: ```suggestion
## Notary Project 
```

> The Notary Project refers to the [GitHub organization](https://github.com/notaryproject), the community, all specifications, and all the repositories under the GitHub organization, including community-released tools like Notary and Notation.

Reviewer: ```suggestion
The Notary Project refers to the [GitHub organization](https://github.com/notaryproject), the community, all specifications, and all the repositories under the GitHub organization, including community-released tools like [notary](https://github.com/notaryproject/notary) and [notation](https://github.com/notaryproject/notation).
```

> ### Notary 

Reviewer 3: ```suggestion
## Notary 
```

> Notary is a TUF [(The Update Framework)](https://theupdateframework/) based artifact signing **implementation**. It consists of a server and client components. The server infrastructure tightly integrates with the container registry and keeps track of the images pushed to the registry. The client component is referred to as the notary command-line interface (CLI), which handles tasks such as signing and pushing container images to a registry and updating metadata. The CLI communicates with the registry and the key and metadata management server component. The most prominent use of this implementation is in Docker Content Trust (DCT). The server and the client implementation can be found in the [notary](https://github.com/notaryproject/notary) repository under the Notary Project organization.

Reviewer 3: ```suggestion
Notary is a TUF [(The Update Framework)](https://theupdateframework.com/) based artifact signing **implementation**. It consists of a server and client components. The server infrastructure tightly integrates with the container registry and keeps track of the images pushed to the registry. The client component is referred to as the notary command-line interface (CLI), which handles tasks such as signing and pushing container images to a registry and updating metadata. The CLI communicates with the registry and the key and metadata management server component. The most prominent use of this implementation is in Docker Content Trust (DCT). 
```

> ### Notation

Reviewer 3: ```suggestion
## Notation
```

Notation, based on new set of specifications unrelated to TUF, is a newer implementation from the Notary Project. It refers to the implementation of the Notation CLI and the Notation libraries. Notation offers signing and verification workflows for OCI artifacts, enables signature portability across OCI-compliant registries, and facilitates integration with third-party key management solutions through a plugin model. The CLI itself is referred to as the [Notation CLI](https://github.com/notaryproject/notation) while the libraries are referred to as the [Notation-Go](https://github.com/notaryproject/notation-go) library and the [Notation-Core-Go](https://github.com/notaryproject/notation-core-go) library. Stakeholders can use libraries independently of the Notation CLI for integrating Notation with their own client tools.

> ### Notary Project Specifications

Reviewer 3: ```suggestion
## Notary Project Specifications
```

> Notary Project Specifications, mainly created between years 2021-2022, are a collection of specifications distributed across "subprojects" within the Notary Project. Notation CLI and libraries have implemented these specifications, which  are also utilized by other OSS projects and/or vendor tools that seek to interoperate with the Notary project tooling. These specifications are stored in the [notaryproject](https://github.com/notaryproject/specifications) repository under the Notary Project.

Reviewer: ```suggestion
Notary Project Specifications, mainly created between years 2021-2022, are a collection of specifications distributed across "subprojects" within the Notary Project. Notation CLI and libraries have implemented these specifications, which are also utilized by other OSS projects and/or vendor tools that seek to interoperate with the Notary Project tooling. These specifications are stored in the [specifications](https://github.com/notaryproject/specifications) repository under the Notary Project.
```

Notary Project specifications are categorized into:

**OCI Signature Specifications:**
These specifications are specific to the [Open Container Initiative (OCI)](https://github.com/opencontainers). It describes how the Notary Project's Signing Scheme is applied to signatures stored in OCI registries. This category includes the following specifications:

> - [Notary Project OCI Signature Specification](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-specification.md)

Reviewer 4: This repo has been renamed to specifications.

Author: Thanks for the update @sajayantony 
The URLs are updated.

- [Notary Project OCI Signing and Verification Workflow](https://github.com/notaryproject/notaryproject/blob/main/specs/signing-and-verification-workflow.md)
- [Notary Project OCI COSE Envelope](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-envelope-cose.md)
- [Notary Project OCI JWS Envelope](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-envelope-jws.md)

**Signing Scheme Specification:** The Signing Scheme Specification is not specific to any particular storage method. It describes the supported signing schemes. Any tool that supports those schemes might be able to produce signatures that are compatible with Notary Project tools. The specification under this category includes the [Notary Project Signing Scheme.](https://github.com/notaryproject/notaryproject/blob/main/specs/signing-scheme.md)

> ### TUF

Reviewer 3: ```suggestion
## TUF
```

---
title: Glossary
description: A list of Notary Project's terminologies 
weight: 11
---
### Notary 

> Notary is a TUF [(The Update Framework)](https://theupdateframework/) based artifact signing solution. It consists of both server and client components. The server infrastructure tightly integrates with the container registry and keeps track of the images pushed to the registry. The client component is referred to as the notary command-line interface (CLI), which handles tasks such as signing and pushing container images to a registry and updating metadata. The CLI communicates with the registry and the key and metadata management server component. The most prominent use of this implementation is in Docker Content Trust (DCT). The server and the client implementation can be found in the [notary](https://github.com/notaryproject/notary) repository under the Notary Project organization.

Reviewer 5: ```suggestion
Notary is a TUF [(The Update Framework)](https://theupdateframework/) based artifact signing **implementation**. It consists of a server and client components. The server infrastructure tightly integrates with the container registry and keeps track of the images pushed to the registry. The client component is referred to as the notary command-line interface (CLI), which handles tasks such as signing and pushing container images to a registry and updating metadata. The CLI communicates with the registry and the key and metadata management server component. The most prominent use of this implementation is in Docker Content Trust (DCT). The server and the client implementation can be found in the [notary](https://github.com/notaryproject/notary) repository under the Notary Project organization.
```

### Notation

> Notation is a non-TUF-based artifact signing solution. It refers to the implementation of the Notation CLI and the Notation libraries. Notation offers signing and verification workflows for OCI artifacts, enables signature portability across OCI-compliant registries, and facilitates integration with third-party key management solutions through a plugin model. The CLI itself is referred to as the [Notation CLI](https://github.com/notaryproject/notation) while the libraries are referred to as the [Notation-Go](https://github.com/notaryproject/notation-go) library and the [Notation-Core-Go](https://github.com/notaryproject/notation-core-go) library. These libraries can be used independently of the Notation CLI.

Reviewer 5: ```suggestion
Notation, based on new set of specifications unrelated to TUF, is a newer implementation from the Notary Project. It refers to the Notation CLI and the Notation libraries. Notation offers signing and verification workflows for OCI artifacts, enables signature portability across OCI-compliant registries, and facilitates integration with third-party key management solutions through a plugin model. The CLI itself is referred to as the [Notation CLI](https://github.com/notaryproject/notation) while the libraries are referred to as the [Notation-Go](https://github.com/notaryproject/notation-go) library and the [Notation-Core-Go](https://github.com/notaryproject/notation-core-go) library. Stakeholders can use libraries independently of the Notation CLI for integrating Notation with their own client tools.
```

> ### Notary Project 

Reviewer 6: Suggest moving Notary Project as the first glossary entry to set overall context and as it is referenced in Notation enrty.

> Notary Project refers to the [GitHub organization](https://github.com/notaryproject), the community, all specifications, and all the repositories under the organization, including community-released tools like Notary and Notation.

Reviewer 5: ```suggestion
The Notary Project refers to the [GitHub organization](https://github.com/notaryproject), the community, all specifications, and all the repositories under the GitHub organization, including community-released tools like Notary and Notation.
```

### Notary Project Specifications

> Notary Project Specifications are a collection of specifications distributed across "subprojects" within the Notary Project. These specifications are also utilized by other OSS projects and/or vendor tools that seek to interoperate with the Notary project tooling. These specifications are stored in the [notaryproject](https://github.com/notaryproject/notaryproject) repository under the Notary Project.

Reviewer 5: ```suggestion
Notary Project Specifications, mainly created between years 2021-2022, are a collection of specifications distributed across "subprojects" within the Notary Project. Notation CLI and libraries have implemented these specifications, which are also utilized by other OSS projects and/or vendor tools that seek to interoperate with the Notary project tooling. These specifications are stored in the [notaryproject](https://github.com/notaryproject/notaryproject) repository under the Notary Project.
```

---
title: Glossary
description: A list of Notary Project's terminologies 
weight: 11
---

> - ### Notary 

Reviewer: Remove `-`

Author: resolve

Notary is a TUF [(The Update Framework)](https://github.com/theupdateframework/specification) based artifact signing solution. It consists of both [server](https://github.com/notaryproject/notary/blob/master/cmd/notary-server) and [client](https://github.com/notaryproject/notary/blob/master/cmd/notary) components for running and interacting with trusted collections. The client component is referred to as the Notary command-line interface (CLI), which handles tasks such as signing and pushing container images to a registry and updating metadata. The CLI communicates with the registry and the key and metadata management server component.

> - ### Notation

Reviewer: Remove `-`

Notation is a non-TUF-based artifact signing solution. It refers to the implementation of the Notation CLI and the Notation Go libraries. Notation offers signing and verification workflows for OCI artifacts, enables signature portability across OCI-compliant registries, and facilitates integration with 3rd party key management solutions through a plugin model. The CLI itself is referred to as the "Notation CLI," while the libraries are referrred to as the [Notation-Go](https://github.com/notaryproject/notation-go) library and the [Notation-Core-Go](https://github.com/notaryproject/notation-core-go) library. These libraries can be used independently of the Notation CLI.

> 

Reviewer: Remove extra line

Author: resolve

> - ### Notary Project Specifications

Reviewer: Remove `-`

- [Notary Project OCI Signature Specification](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-specification.md)
- [Notary Project OCI Signing and Verification Workflow](https://github.com/notaryproject/notaryproject/blob/main/specs/signing-and-verification-workflow.md)
- [Notary Project OCI COSE Envelope](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-envelope-cose.md)
- [Notary Project OCI JWS Envelope](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-envelope-jws.md)

**Signing Scheme Specification:** The Signing Scheme Specification is not specific to any particular storage method. It describes the supported signing schemes. Any tool that supports those schemes might be able to produce signatures that are compatible with Notary Project tools. The specification under this category includes the [Notary Project Signing Scheme.](https://github.com/notaryproject/notaryproject/blob/main/specs/signing-scheme.md)

> 

Reviewer: Remove extra line

Author: resolve

> - ### TUF

Reviewer: Remove `-`

Author: resolve

The Update Framework(TUF) is a project under the Notary Project that aims to implement the full TUF specification in a registry-native way. The initial version of Notary did not support transferring signatures between different registries and did not gain widespread adoption. The TUF project aims to develop an enhanced version of Notary, addressing these limitations and encouraging widespread adoption and usage.

> 

Reviewer: Remove extra line

Author: resolve
